configs/raspberrypi3+_wpe_ml_defconfig

```diff
@@ -56,8 +56,9 @@ BR2_PACKAGE_BITSTREAM_VERA=y
 BR2_PACKAGE_LINUX_FIRMWARE=y
 BR2_PACKAGE_LINUX_FIRMWARE_RALINK_RT2XX=y
 BR2_PACKAGE_RPI_FIRMWARE=y
+BR2_PACKAGE_RPI_VERSION_RPI3PLUS=y
```

Comment:
These flags are defined in the choice section so this situation is not possible. It is the recommended way to go with mutually exclusive choices in buildroot (eg. Glibc version, GCC version & many others)
![screenshot](https://user-images.githubusercontent.com/22924102/73358552-25d26480-429f-11ea-9bfb-a42e12c64729.png)
You could manually edit .config by text editor and set it, but editing .config manually is not recommended. Even if you do that and corrupt the config, menuconfig utillity will spot it and automatically correct it. 

I guess if we made that a stirng it would be more error prone: Do you write "RPI3+", "3+", "3B+"? What version are available? Etc...

Third options would be to made a choice that ends up as a string, but these flags (BR2_PACKAGE_RPI_VERSION_RPI4, BR2_PACKAGE_RPI_VERSION_RPI3, etc) would still have to exist to set the value of BR2_PACKAGE_RPI_VERSION, so it would just end as a duplication

Reply:
Oke, I will merge and do it on my side :-) doing this on this channel takes longer.

Reply:
:/